### PR TITLE
fixes adding missing labels in datetime

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|screen shot 2016-06-28 at 16 52 18|screen shot 2016-06-28 at 16 52 40|

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
 * Fixes input and textarea fields in not aligned inline groups
 * Fixes select field width in table
+* Adding missing labels to datetime
```
Comment (vxsx) [inline CHANGELOG.rst]: i think should be more specific 
`Added missing labels to datetime fields that are already part of the field box`

**Comment by vxsx** [comment]
otherwise LGTM

File: CHANGELOG.rst (hunk 1 above)
Comment (lory87) [inline CHANGELOG.rst]: 👍

**Comment by vxsx** [comment]
👍

**Comment by FinalAngel** [comment]
@lory87 @vxsx can we start adding screenshot tests from now on, on every new PR?

**Comment by lory87** [comment]
@FinalAngel We need an extra app which includes all the custom fields and elements. The changes were done for elements in custom apps. I already talked with @vxsx about this.

**Comment by FinalAngel** [comment]
@mkoistinen or @czpython what would be the efforts to add those custom fields and configurations here?

**Comment by czpython** [comment]
@FinalAngel where would this app live?

**Comment by FinalAngel** [comment]
@czpython we would need to integrate it here, so we can make screenshot tests to compare if we fix / adapt things

**Comment by vxsx** [comment]
@FinalAngel @czpython not necessarily. it can be a completely separate something that is required by the test_requirements, whatever would be easier

**Pull request merged**